README.md

```diff
@@ -157,6 +157,51 @@ model_pb = entity_pb_to_model_pb(my_model_pb2.MyModelPB, entity_pb)
 print(model_pb)
 ```
 
+## Excluding Protobuf Model Fields from Indexes
+
+By default, Google Cloud Datstore automatically indexes each entity (model) property.
+
+Indexing each field (entity property) is usually not desired nor needed. It also has some
+limitations (for example, size of a simple field which is to be indexed is limited to ``1500``
+bytes, etc.). In addition to that, uncessary indexing causes increased storage space consumption.
+
+This library allows you to define which model fields to exclude from index on the field basis
+utilizing Protobuf field options extension.
+
+For example:
+
+```protobuf
+syntax = "proto3";
+
+import "google/protobuf/descriptor.proto";
+
+// Custom Protobuf option which specifies which model fields should be excluded
+// from index
+// NOTE: Keep in mind that it's important not to change the option name
+// ("exclude_from_index") since this library uses that special option name to
+// determine if a field should be excluded from index.
+extend google.protobuf.FieldOptions {
+    bool exclude_from_index = 5000;
```

Comment:
Hi, any particular reason to use 5000 as Filed Number? Cheers

Reply:
Yeah, as @Sheshagiri said - I wanted to use ``50000``.

In short it can be any (high) number which is unlikely to clash with other user defined fields and extensions.

In this case it matters even less because I find the extension based on the name (``exclude_from_index``) and not based on the number.

On the naming side - I was also thinking of prefixing the extension name with ``translator.`` or similar, to potentially avoid conflicts with any other extensions which may use the same name, but I don't think it's a huge deal since this translator library is pretty specific and it's very unlikely that user would use some other custom extension which uses the same name.